_sass/_hackathon.scss

```diff
@@ -0,0 +1,24 @@
+#csh-hacks-content{
+  padding-top: 10px;
+  padding-bottom: 10px;
```

Comment:
Why have an inner ID for something that the `-wrapper` class can already achieve? Look to the `#homepage-about` section for guidance imo. I don't think there's any need to have styles on the content and the wrapper, as they're both just `div`s.